### DISCUSSION TITLE
NEED API UPDATE GUIDE

### DISCUSSION THREAD

**Discussion by Frozia00** [kotatogram/kotatogram-desktop#446] [EN 1. General]
Is there a way to update Kotatogram's API, and is there a comprehensive guide on this matter? How can we achieve this? We are unable to utilize Telegram's current updated features.

**Comment by ilya-fedin** [comment]
You can't really update supported API version without major changes which are more complicated than merging new tdesktop sources sadly... Merging tdesktop sources is not simple as well, requires way more time than @EricKotato has, sadly. You can try the  patches-track* in-progress branches if you want

**Reply by ilya-fedin** [reply]
I'm not sure how Telegram X is related given that it's an Android client while Kotatogram is a desktop client?
patches-track branches are git branches. If you aren't a developer and you don't know how to build applications from source, that perhaps isn't helpful to you.
